tests/storages_tests/rdb_tests/test_storage.py

```diff
@@ -286,3 +301,63 @@ def test_get_trials_excluded_trial_ids() -> None:
     # See https://github.com/optuna/optuna/issues/1457.
     trials = storage._get_trials(study_id, states=None, excluded_trial_ids=set(range(500000)))
     assert len(trials) == 0
+
+
+def test_record_heartbeat() -> None:
+    heartbeat_interval = 1
+    n_trials = 2
+
+    def objective(trial: Trial) -> float:
+        time.sleep(2)
+        return 1.0
+
+    storage = create_test_storage(heartbeat_interval=heartbeat_interval)
+    assert isinstance(storage, RDBStorage)
+    study = create_study(storage=storage)
```

Comment:
[IMO]

In the current implementation, users have to create storage themselves.
How about adding `heartbeat_interval` to `crete_study` so that users can specify the URI of the database?

```python
study = optuna.load_study(
    study_name="distributed-example",
    storage="mysql://root@localhost/example",
    heartbeat_interval=1,
)

study.optimize(objective, n_trials=100)
```

Reply:
Thanks for the careful consideration. As you mentioned, I initially added the `heartbeat_interval` argument on the `create_study`, but I've removed it as suggested by @hvy in https://github.com/optuna/optuna/pull/2186#discussion_r555484797. I think either is fine to add `heartbeat_interval` on storages or the `create_study`. I'd like to know @hvy's opinion.

Reply:
Discussed this elsewhere so leaving a record but we decided to keep this "private" for now. It's experimental to some extent.